test/fnlb-test-harness/primes-func/func.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"strconv"
+	"time"
+)
+
+// return list of primes less than N
+// source: http://stackoverflow.com/a/21923233
+func sieveOfEratosthenes(N int) (primes []int) {
+	b := make([]bool, N)
+	for i := 2; i < N; i++ {
+		if b[i] == true {
+			continue
+		}
+		primes = append(primes, i)
+		for k := i * i; k < N; k += i {
+			b[k] = true
+		}
+	}
+	return
+}
+
+func main() {
+	start := time.Now()
+	maxPrime := 1000000
+	numLoops := 1
+
+	// Parse the query string
+	s := strings.Split(os.Getenv("REQUEST_URL"), "?")
+	if len(s) > 1 {
+		for _, pair := range strings.Split(s[1], "&") {
+			kv := strings.Split(pair, "=")
+			if len(kv) > 1 {
+				key, value := kv[0], kv[1]
+				if key == "max" {
+					maxPrime, _ = strconv.Atoi(value)
+				}
+				if key == "loops" {
+					numLoops, _ = strconv.Atoi(value)
+				}
+			}
+		}
+	}
+
+	for i := 0; i < numLoops; i++ {
```

Comment:
I am not sure what this part is for honestly nothing is changing in this for loop